File: NonBlockingAsync/PlayJava/ViewhistoryService/app/model/Item.java
package model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter @AllArgsConstructor
public class Item {
    private String id;
    private String name;
    private String description;
    private String imageUrl;
    private double price;
    private Integer customerID;
}

File: NonBlockingAsync/NetCore/eBookStore/Models/Item.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace eBookStore.Models
{
    public class Item
    {
        public string id { get; set; }
        public string name { get; set; }
        public string imageUrl { get; set; }
        public double price {get; set;}
        public int customerID { get; set; }

    }
}
